app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -404,14 +404,9 @@ def save_custom_attribute_attribute_inventory(entity, attribute_name, hashes, ta
     return if hashes.nil?
 
     entity.send(attribute_name).reset
-    deletes = if target.kind_of?(ExtManagementSystem)
-                :use_association
-              else
-                []
-              end
 
     save_inventory_multi(entity.send(attribute_name),
-                         hashes, deletes, [:section, :name])
+                         hashes, :use_association, [:section, :name])
```

Comment:
If you don't want to move these functions to save_inventory.rb, please add a comment that they're also serving Amazon refresh.
(The move should be a trivial textual move, there is nothing else to refactor.  Again, these modules all end up in one namespace.)